### ISSUE TITLE
URL for latest stable release (2.4.3.0) is not valid

### ISSUE THREAD

**Issue by issaclin32** [rcmaehl/WhyNotWin11#674]
[URL for latest stable release](https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe) (version 2.4.3.0) is not valid

https://github.com/rcmaehl/WhyNotWin11#readme
```
Download
[Download latest stable release](https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe)

[Download latest testing release](https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip)
```

**Comment by rcmaehl** [comment]
... I could have sworn there was a link at one point.

**Comment by rcmaehl** [comment]
Fixed for 2.4.3.2
